src/App.jsx

```diff
@@ -1,80 +1,16 @@
 import React from 'react';
 import './App.css';
+import Article from './components/Article/Article';
+import Header from './components/Header/Header';
+import Welcome from './components/Welcome/Welcome';
```

Comment:
if you create index file for each of components - the import statements would be shorter
```suggestion
import Article from './components/Article';
import Header from './components/Header';
import Welcome from './components/Welcome';
```